install_sct

```diff
@@ -57,8 +91,26 @@ for installation instructions check: https://git-scm.com/book/en/v2/Getting-Star
 "
   fi
 
+  if [[ ! $(which gcc) ]]; then
```

Comment:
this is not a sensitive test. For example, on an OSX where gcc is not installed (but still "recognized"), I get this:
~~~
$ which gcc
/usr/bin/gcc
$ gcc
** INTERNAL ERROR: Uncaught exception **
Uncaught Exception: Error getting value for key 'initializationClass' of extension 'Xcode.DVTKitDFRSupport.Initializer' in plug-in 'com.apple.dt.IDE.DVTKitDFRSupport'
Stack:
  0   __exceptionPreprocess (in CoreFoundation)
  1   objc_exception_throw (in libobjc.A.dylib)
  2   -[DVTExtension valueForKey:error:] (in DVTFoundation)
  3   _IDEInitializeOnePlugInAndPrerequisites (in IDEFoundation)
  4   _IDEInitializePlugIns (in IDEFoundation)
  5   IDEInitialize (in IDEFoundation)
  6   -[Xcode3CommandLineBuildTool run] (in Xcode3Core)
  7  0x00000001083b6202 (in xcodebuild)
  8   start (in libdyld.dylib)

sh: line 1:  3211 Abort trap: 6           /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -sdk macosx -find gcc 2> /dev/null
gcc: error: unable to find utility "gcc", not a developer tool or in PATH
~~~

Reply:
nevermind-- your code works (because output status is not 0)

Reply:
actually, when running the actual installer, it does **not** work ;-)

on an OSX that has missing gcc (see message above), the installer fails, giving:
~~~
...
    
    sh: line 1:  4420 Abort trap: 6           /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -sdk macosx -find gcc 2> /dev/null
    gcc: error: unable to find utility "gcc", not a developer tool or in PATH
    error: command 'gcc' failed with exit status 72
    ----------------------------------------
ERROR: Command "/Users/jcohen/sct-dev2/python/envs/venv_sct/bin/python -u -c 'import setuptools, tokenize;__file__='"'"'/private/var/folders/r3/9skf4b5x4y14t1jh0zs0g0l80000gn/T/pip-install-757cu9wr/psutil/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/r3/9skf4b5x4y14t1jh0zs0g0l80000gn/T/pip-record-uxgqf44l/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /private/var/folders/r3/9skf4b5x4y14t1jh0zs0g0l80000gn/T/pip-install-757cu9wr/psutil/

Failed running pip install -r requirements.txt: 0
~~~

Reply:
which means that the `if [[ ! $(which gcc) ]]; then` clause was not effective

Reply:
ok, i get it. To be effective, we would have to do something like:
~~~
$gcc --version
$ echo $?
72  # --> GCC NOT WORKING
~~~